### DISCUSSION TITLE
Install chezmoi on Rocky Linux

### DISCUSSION THREAD

**Discussion by halloleo** [twpayne/chezmoi#3909] [Q&A] [answered]
I want to install chezmoi on Rocky Linux 8 (CentOS successor).

Is there for `yum`/`dnf` a package repo available which hosts chezmoi?

I had a look on [the install page](https://www.chezmoi.io/install/), but couldn't find anything for RHEL/CentOS/Rocky.

Thanks for pointers!

**Comment by twpayne** [comment, marked as answer]
Look at https://www.chezmoi.io/install/#download-a-pre-built-linux-package.

**Reply by halloleo** [reply]
Brilliant! Thanks heaps. Best solution I'd say. (I was pondering Homebrew for Linux, but worried it might be overkill.)
